File: ifHolidayQuit.js
// open file
var fs = require('fs');

function checkHoliday(a,b,callback){
  fs.readFile('/home/ec2-user/stack/isHoliday.csv', 'utf8', function(err, data) {
      if (err) throw err;
      console.log(data);
      if (data != "true"){
        console.log("Continuing, today is not a holiday.");
      }else{
        console.log("Quitting, today is a holiday.");
        process.exit();
      }
  })};

// --------------------------------------------------------------------------------------------
// Run in a fiber -----------------------------------------------------------------------------
var Sync = require('sync');
Sync(function(){
     try {
          var resCheckHoliday = checkHoliday.sync(null,null,null);
     }
     catch (e) {
      console.error(e);
     }
})

File: ifHolidayQuit.py
# -*- coding: utf-8 -*-
"""
Created on Sun Jan 14 14:52:05 2018

@author: micah
"""
import sys

file = open("/home/ec2-user/stack/isHoliday.csv", "r") 
#print file.read()
if file.read() == "true":
    sys.exit("Quitting because today is a holiday.")
else:
    print("Continuing because today is not a holiday.")
